src/map/utils/fellowutils.h

```diff
@@ -0,0 +1,116 @@
+﻿/*
+===========================================================================
+
+  Copyright (c) 2010-2015 Darkstar Dev Teams
```

Comment:
If you've used someone else's work as reference, make sure they're correctly attributed (Presumably this'll be Wiggo and KO)

https://github.com/LandSandBoat/server/wiki/Development-Guide-Git#git-attribution

Reply:
Good call, had meant to credit Wiggo here and will update the commit.
I know KO has done a lot of fellow work on Eden but this is sourced from Wings, which I implemented using Wiggos original LSB PR.

Reply:
Added links to source in PR overview - reached out to Wiggo32 and awaiting confirmation of email address to use for the commit.

Reply:
Per Wiggo32 today - there is not a good email to use for attribution.
Added as a co-author w/o email.